arlo-client/src/components/DataEntry/index.tsx

```diff
@@ -212,16 +230,25 @@ const DataEntry: React.FC<IProps> = ({
   }
 
   const submitBallot = (batchId: string, ballotPosition: number) => async (
-    interpretations: IBallotInterpretation[]
+    interpretations?: IBallotInterpretation[]
```

Comment:
this feels slightly tricky to me because there is a big difference in meaning between `interpretations` being set to `[]` vs `null` or `undefined`. is there a clearer interface?

Reply:
in this case I wanted to reduce internal API complexity so felt this was the simplest smallest change.